src/transaction.ts

```diff
@@ -1568,6 +1620,12 @@ export class Transaction extends Dml {
       reqOpts.returnCommitStats = (options as CommitOptions).returnCommitStats;
     }
 
+    reqOpts.requestOptions = this.configureTagOptions(
+      typeof reqOpts.singleUseTransaction !== 'undefined',
+      this.requestOptions?.transactionTag!,
+      requestOptions
+    );
+
     this.request(
       {
         client: 'SpannerClient',
```

Comment:
The `PartitionedDml` class at the end of this file should support setting a `requestTag`.

Reply:
`requestTag` should be able to be set at function level, `runUpdate` accepts `requestTag` as part of the `ExecuteSqlRequest.requestOptions`